Tests/DangerTests/GitDiffTests.swift

```diff
@@ -143,6 +143,8 @@ final class GitDiffTests: XCTestCase {
         ]))
     }
 
+    // swiftlint:disable all
+    // swiftformat:disable all
```

Comment:
``` swift
// swiftlint:disable trailing_whitespace
// swiftformat:disable trimwhitespace
```

I could't get any effect by using this set of disable so I turn to use all instead in the end.

Reply:
I will have a look at this separately, probably will just use a swiftformat config

Reply:
yeah you are right, that might be the better way to solve this kind of issues!